docs/maps/vector-tooltips.asciidoc

```diff
@@ -6,8 +6,8 @@ These tooltips give users an in-depth insight into what's going on in the map.
 
 If more than one feature exists at a location, the tooltip displays the
 attributes for the top feature, and notes the number of features at that location.
-The following image shows a tooltip with three features at the current location.
-The tooltip displays attributes for the top feature, the green circle.
+The following image shows a tooltip with three features at the current location. A green circle from the *Total Sales Revenue* layer, the blue New York State polygon from *United States* layer, and the red United States Country polygon from the *World Countries* layer.
```

Comment:
```suggestion
The following image has a tooltip with three features at the current location: a green circle from the *Total Sales Revenue* layer, a blue New York State polygon from *United States* layer, and a red United States Country polygon from the *World Countries* layer.
```

Reply:
The word "show" in the image above made me think that I would see all three icons in the image.